src/components/pagination/pager-page.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {Fragment, useEffect} from "react";
+
+export interface PagerPageProps {
+    pageIndex: number
+    numberOfElementsPerPage: number
+    onLastPageIndexChange: (lastPageIndex: number) => void
+}
+
+export const PagerPage: React.FC<PagerPageProps> = ({children, numberOfElementsPerPage, pageIndex, onLastPageIndexChange}) => {
+
+    useEffect(() => {
+        const lastPageIndex = Math.ceil(React.Children.count(children) / numberOfElementsPerPage) - 1;
+        onLastPageIndexChange(lastPageIndex)
+    }, [children, numberOfElementsPerPage, onLastPageIndexChange])
```

Comment:
```suggestion
    const lastPageIndex = Math.ceil(React.Children.count(children) / numberOfElementsPerPage) - 1;
    useEffect(() => {
        onLastPageIndexChange(lastPageIndex)
    }, [lastPageIndex])
```

Reply:
Why? I don't need this value outside of the effect.

Reply:
then it should stay like that. My mistake